runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorker.java

```diff
@@ -726,6 +726,7 @@ public void setRetryLocallyDelayMs(int retryLocallyDelayMs) {
 
   @VisibleForTesting
   public void setMaxWorkItemCommitBytes(int maxWorkItemCommitBytes) {
+    LOG.info("Setting maxWorkItemCommitBytes to {}", maxWorkItemCommitBytes);
```

Comment:
only log if the value changes, otherwise I think we poll the global config and this log would happen regularly.

otherwise lgtm

Reply:
Done.